package.json

```diff
@@ -21,29 +22,40 @@
     "build": "yarn workspaces run build",
     "build:contracts": "yarn workspace @clrfund/contracts run build",
     "build:web": "yarn workspace @clrfund/vue-app run build",
+    "build:subgraph": "yarn workspace @clrfund/subgraph run codegen && yarn workspace @clrfund/subgraph run build",
```

Comment:
Should this be part of `yarn start:dev` by default? Do we need to redeploy a subgraph every time we boot up our local env? Or can we keep the docker container running & just wipe the db?

Reply:
Yes, that could be a good idea, thanks for the suggestion. First you turn on the two nodes and then you just `start:dev`.

Will test it and make it but I'll add that in a different PR.